### PR TITLE
Customize api server port

### PR DESCRIPTION
**Pull request by StrongMonkey**

### CODE REVIEW

**Review by brandond — APPROVED** [review]

**Comment by brandond** [comment]
LMK if you are unable to merge

**Comment by StrongMonkey** [comment]
@brandond yeah I don't have permission to merge this

File: pkg/controllers/chart/chart.go [hunk 1]
```diff
@@ -68,10 +68,11 @@ type Controller struct {
 	jobCache        batchcontroller.JobCache
 	apply           apply.Apply
 	recorder        record.EventRecorder
+	apiServerPort   string
 }
 
 func Register(ctx context.Context,
-	systemNamespace, managedBy string,
+	systemNamespace, managedBy string, apiServerPort string,
```
Comment (brandond) [inline pkg/controllers/chart/chart.go:75]:
```suggestion
	systemNamespace, managedBy, apiServerPort string,
```

**Review by brandond — CHANGES_REQUESTED** [review]
nit on function args

Will merge and tag once resolved.

File: pkg/controllers/chart/chart.go (hunk 1 above)
Comment (StrongMonkey) [reply pkg/controllers/chart/chart.go]: updated

**Review by brandond — APPROVED** [review]

**Pull request merged**